recipe/download_osx_sdk.sh

```diff
@@ -24,29 +24,42 @@ fi
 export CONDA_BUILD_SYSROOT="${OSX_SDK_DIR}/MacOSX${MACOSX_SDK_VERSION}.sdk"
```

Comment:
If we only use major versions, local copies might stay in older minor releases. Not sure if we want or need to do something about it.

Reply:
Changed this to use the actual version instead.